packages/stack/src/navigators/createStackNavigator.tsx

```diff
@@ -94,3 +96,13 @@ export default createNavigatorFactory<
   StackNavigationEventMap,
   typeof StackNavigator
 >(StackNavigator);
+
+export type TypedStackNavigator<
```

Comment:
Can you rename it to `StackNavigatorType`?

Reply:
Should I also rename `TypedNavigator` to `NavigatorType`? (That's why I named it `TypedStackNavigator`, because `TypedNavigator` already existed.)

Reply:
That's internal, so it doesn't matter what we name it. I think `TypedNavigator` is ok for internal use.